Gemfile

```diff
@@ -20,6 +18,10 @@ gem "minima", "~> 2.0"
 # If you have any plugins, put them here!
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.6"
+  gem "jekyll-paginate"
```

Comment:
Run `bundle update` and use [pessimize](https://github.com/joonty/pessimize#installation) to add Semantic Versioning restriction to all gems for future proofing.